src/constants.ts

```diff
@@ -0,0 +1,7 @@
+export default {
+  blocktime: 4,
+  numOfDelegates: 51,
+  messageLengthLimit: 512,
```

Comment:
Make this configurable either via an `.env` file.

Reply:
https://github.com/ArkEcosystem/poc-ark-messenger/pull/7